src/lib/Form/DataTransformer/ContentInfoTransformer.php

```diff
@@ -53,29 +40,18 @@ public function transform($value)
         return $value->id;
     }
 
-    /**
-     * Transforms a Content's ID integer into a domain specific ContentInfo object.
-     *
-     * @param mixed|null $value
-     *
-     * @return ContentInfo|null
-     *
-     * @throws UnauthorizedException
-     * @throws TransformationFailedException if the given value is not an integer
-     *                                       or if the value can not be transformed
-     */
     public function reverseTransform($value): ?ContentInfo
```

Comment:
I think in those terms it make sense to leave typehinted $value in docs. Just not mixed ;)

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1148/commits/42437d627f79cccd4225b508580a7512e00f0d8d